bluetooth_mesh/application.py

```diff
@@ -918,22 +933,29 @@ def _read_message() -> Any:
     async def import_node(
         self,
         join_callback: Optional[Callable[[int], Awaitable[int]]] = None,
-        flags: Optional[Mapping[str, Any]] = None,
+        key_refresh: bool = False,
     ) -> int:
         """
         Create a self-provisioned node.
         """
         net_index, net_key = self.primary_net_key
 
-        self.logger.warning("Import node %s, address %04x, iv index %i", self.uuid.hex, self.address, self.iv_index)
+        self.logger.warning(
+            f"Import node %s, address %04x, iv index %i{'(updating)' if self.iv_update else ''}",
```

Comment:
Ewwwwww. Do it outside of the string please.

Reply:
Fixed